project-management/1_to_1_Sprint_User_Review_sessions_guidelines.md

```diff
@@ -0,0 +1,32 @@
+# 1 to 1 Sprint User Review 
+
+A 1 to 1 review of each Sprint’s activity will be, completed at the end of each Sprint.  
+With one member of the Fujitsu Team and one member of the Autistic community chosen by Autistica. 
+The review meetings can be undertaken via skype / or as a phone call with shared screens or in person dependant on the expressed preference of the member from the Autistic Community. 
+Autistica will source the member from the autistic community and ensure all consent required by Alan Turing is obtained. The community members will be selected from as broad a cross-section of the community as possible. As a result, the review process will be accommodating to their needs. 
+Autistica will give notice to Fujitsu in advance of who will be attending each 1 to 1 and how they would prefer for it to be, carried out to allow adequate time to plan. This will be in the form of a member booking form. (doc to be created) 
+
+## Prior to Session:
+
+The members of the autistic community should watch demonstration videos & look at the sprint review report from previous sprints to be up to date with project and what is to be asked of them.
+Autistica will ensure that all volunteers understand that this is a pre-requisite to involvement in the project, and will take responsibility to inform volunteers.
```

Comment:
I'm uncomfortable with this language. Please revise to better communicate our understanding of people's personal needs and requirements.

Reply:
I've made a change, please let me know if this suffices.